### PR TITLE
gaming

### PR DESCRIPTION
**Pull request by Boland-Unfug**

### CODE REVIEW

**Comment by Boland-Unfug** [comment]
I can make comments@!

File: plantTemp.cpp [hunk 1]
```diff
@@ -1,7 +1,10 @@
 #include <iostream>
 #include <cmath>
 using namespace std;
-
+//gaming
```
Comment (Omnatth) [inline plantTemp.cpp:4]: gaming

**Comment by Boland-Unfug** [comment]
maybe its through this?

File: plantTemp.cpp (hunk 1 above)
Comment (Boland-Unfug) [reply plantTemp.cpp:4]: gaming

**Review by Omnatth — APPROVED** [review]

**Pull request merged**